ui/admin/tests/acceptance/onboarding/success-test.js

```diff
@@ -4,10 +4,12 @@
  */
 
 import { module, test } from 'qunit';
-import { visit, currentURL, find, click, fillIn } from '@ember/test-helpers';
+import { visit, currentURL, click, fillIn } from '@ember/test-helpers';
 import { setupApplicationTest } from 'admin/tests/helpers';
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 import { authenticateSession } from 'ember-simple-auth/test-support';
+import * as selectors from './selectors';
+import * as commonSelectors from 'admin/tests/helpers/selectors';
 
 module('Acceptance | onboarding | success', function (hooks) {
```

Comment:
Is there a reason why `doneButtonSelector` its not refactored into the selectors file? I knwo was there before and perhaps not following any previous pattern, but I think it can be moved to `onboarding/selectors.js`, let me know your thoughts 😉

Reply:
Nice catch Carlos! I didn't notice that in my review.